src/main/java/bio/terra/service/snapshotbuilder/utils/QueryBuilderFactory.java

```diff
@@ -5,6 +5,11 @@
 
 @Component
 public class QueryBuilderFactory {
+
+  public static final String HAS_CHILDREN = "has_children";
```

Comment:
Putting the aliases inside `QueryBuilderFactory` was a suggestion, thoughts on making a new file for these?

Reply:
I don't have a strong opinion on whether they live here or in some alias file

Reply:
I think this is a good enough place for now.